qiskit/circuit/library/standard_gates/s.py

```diff
@@ -78,6 +82,10 @@ def __array__(self, dtype=None):
         """Return a numpy.array for the S gate."""
         return numpy.array([[1, 0], [0, 1j]], dtype=dtype)
 
+    def power(self, exponent: float):
+        """Raise gate to a power."""
+        return PhaseGate(0.5 * numpy.pi * exponent)
+
 
 class SdgGate(Gate):
```

Comment:
Should we also override the method here? 🙂

Reply:
Done.